.github/workflows/static_analysis.yaml

```diff
@@ -123,7 +124,7 @@ jobs:
           python3 -mvenv venv
           venv/bin/pip3 install -r requirements.txt
           venv/bin/python3 cmake_downloader.py --first_minor
-          output=$(venv/bin/python3 cmake_min_version.py /home/runner/work/librealsense/librealsense/)
+          output=$(venv/bin/python3 cmake_min_version.py $work_dir)
```

Comment:
Please add a check after `current_cmake_version=` line that current_cmake_version is in the format of
dd.dd.dd.dd (you can use regex)

Reply:
When it didnt work it contained an error string

and < can return true on that case

Reply:
> and < can return true on that case

May you explain what you mean here?

Reply:
> Please add a check after `current_cmake_version=` line that current_cmake_version is in the format of dd.dd.dd.dd (you can use regex)

Implemented.

Reply:
> here

I mean this is probably result true
if "3.8.0" < "cannot find version"

Reply:
> > here
> 
> I mean this is probably result true if "3.8.0" < "cannot find version"

I checked it and you right - there can be bug.
I added exit case before this `if`